src/data/Projects/projectsDescriptionList.ts

```diff
@@ -0,0 +1,69 @@
+// tslint:disable
+import {ProjectInterface} from "./ProjectInterface";
+
+const currentDate = new Date();
+
+export const projectsList: ProjectInterface [] = [
```

Comment:
This file must be json. 
Dates must have format "Y-m-d"
Descriptions should be array